src/orion/core/worker/storage_backup.py

```diff
@@ -0,0 +1,725 @@
+# pylint: disable=,protected-access,too-many-locals,too-many-branches,too-many-statements
+"""
```

Comment:
This module should be in `orion.storage` instead.

Reply:
Done: https://github.com/Epistimio/orion/pull/1082/commits/5fe487b6be09904df125fe9e16f54d337ef7a3ab